.eslintrc.json

```diff
@@ -0,0 +1,6 @@
+{
+    "extends": "eslint-config-edx",
+    "globals": {
+        "setFixtures": true  // added globally by jasmine-jquery (bad)
```

Comment:
This looks like it's from the ui-toolkit, do we need `setFixtures` to be global in this repo? I don't think we do the kind of tests that would use it here.